app/db/models/application_config.py

```diff
@@ -0,0 +1,195 @@
+    template_name = Column("Template Name", String(), nullable=True)
+    is_template = Column("is_template", Boolean, default=False, nullable=False)
+    audit_info = Column("audit_info", JSON(none_as_null=True))
+    section_index = Column(Integer())
+    pages: Mapped[List["Page"]] = relationship(
+        "Page", order_by="Page.form_index", collection_class=ordering_list("form_index")
+    )
+    runner_publish_name = Column(db.String())
+    source_template_id = Column(UUID(as_uuid=True), nullable=True)
+
+    def __repr__(self):
+        return f"Form({self.runner_publish_name} - {self.name_in_apply['en']}, Pages: {self.pages})"
+
+
+@dataclass
+class Page(BaseModel):
+
+    form_id = Column(
+        "form_id",
+        UUID(as_uuid=True),
+        ForeignKey("form.form_id"),
+        nullable=True,  # will be null where this is a template and not linked to a form
+    )
+    page_id = Column(
+        "page_id",
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid4,
+    )
+    name_in_apply = Column("name_in_apply_json", JSON(none_as_null=True), nullable=False, unique=False)
+    template_name = Column("Template Name", String(), nullable=True)
+    is_template = Column("is_template", Boolean, default=False, nullable=False)
+    audit_info = Column("audit_info", JSON(none_as_null=True))
+    form_index = Column(Integer())
+    display_path = Column("display_path", String())
+    components: Mapped[List["Component"]] = relationship(
+        "Component", order_by="Component.page_index", collection_class=ordering_list("page_index")
+    )
+    source_template_id = Column(UUID(as_uuid=True), nullable=True)
+
+    def __repr__(self):
+        return f"Page(/{self.display_path} - {self.name_in_apply['en']}, Components: {self.components})"
+
+
+class Lizt(BaseModel):
+    list_id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid4,
+    )
+    name = Column(String())
+    type = Column(String())
+    items = Column(JSON())
+
+
+@dataclass
+class Component(BaseModel):
+
+    component_id = Column(
+        "component_id",
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid4,
+    )
+    page_id = Column(
+        "page_id",
+        UUID(as_uuid=True),
+        ForeignKey("page.page_id"),
+        nullable=True,  # will be null where this is a template and not linked to a page
+    )
+    theme_id = Column(
+        UUID(as_uuid=True),
+        ForeignKey("theme.theme_id"),
+        nullable=True,  # will be null where this is a template and not linked to a theme
+    )
+    # TODO make these 2 json so we can do welsh?
+    title = Column(String())
+    hint_text = Column(String(), nullable=True)
+    options = Column(JSON(none_as_null=False))
+    type = Column(ENUM(ComponentType))
+    template_name = Column("Template Name", String(), nullable=True)
```

Comment:
I thought we might of had a conflict with the Third Normal form here
**
Transitive Dependency: This occurs when one non-key column is functionally dependent on another non-key column, which itself is dependent on the primary key.**

I dont think this is the case though, as everything appears to point back to the PK (unless I have misunderstood).

Reply:
Yeah this is just a 'display name' for when it's a template